BuildResidentialHPXML/measure.rb

```diff
@@ -4845,6 +5151,14 @@ def self.set_building_header(hpxml_bldg, args)
       end
       hpxml_bldg.header.extension_properties = extension_properties
     end
+
+    if not args[:electric_panel_baseline_peak_electricity_power].nil?
+      if hpxml_bldg.header.extension_properties.nil?
+        hpxml_bldg.header.extension_properties = { 'PeakElectricity' => args[:electric_panel_baseline_peak_electricity_power] }
+      else
+        hpxml_bldg.header.extension_properties['PeakElectricity'] = args[:electric_panel_baseline_peak_electricity_power]
+      end
+    end
```

Comment:
We shouldn't be using the `extension_properties`, that's for misc stuff that OS-HPXML doesn't know anything about. Maybe it should be a new extension element under `BuildingSummary`? It could also use a better name that gives some context.

Also, this input needs to be documented and added to the schematron validator. It's currently completely unknown to any user that doesn't use the BuildResHPXML measure.

Reply:
Moved to `/HPXML/Building/BuildingDetails/BuildingSummary/extension/ElectricPanelBaselinePeakPower`.